package.json

```diff
@@ -14,7 +14,7 @@
     "lint-autofix": "tslint -p . -c tslint.json -t verbose --fix",
     "generate:mock-google-services-json": "cp mock-google-services.json ./android/app/google-services.json",
     "generate:pagopa-api-definitions": "rimraf definitions/pagopa && mkdir -p definitions/pagopa && gen-api-models --api-spec https://wisp2.pagopa.gov.it/pp-restapi-CD/v2/api-docs --out-dir ./definitions/pagopa --no-strict --request-types --response-decoders --default-error-type Error",
-    "generate:api-definitions": "rimraf definitions/backend && mkdir -p definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v1.0.1/api_backend.yaml --out-dir ./definitions/backend --no-strict --request-types --response-decoders  && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v1.0.1/api_public.yaml --out-dir ./definitions/backend",
+    "generate:api-definitions": "rimraf definitions/backend && mkdir -p definitions/backend && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/163190244-tos-implementation/api_backend.yaml --out-dir ./definitions/backend --no-strict --request-types --response-decoders  && gen-api-models --api-spec https://raw.githubusercontent.com/teamdigitale/italia-backend/v1.0.0/api_public.yaml --out-dir ./definitions/backend",
     "bundle:android-release": "node node_modules/react-native/local-cli/cli.js bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/",
```

Comment:
italia-backend/v1.0.1

Reply:
latest is actually `v1.1.0`

Reply:
The version had been updated (even if the preview here shows it points to the tos branch)